extensions/wikia/WikiaApi/WikiaApiQueryAllUsers.php

```diff
@@ -170,89 +158,89 @@ private function global_users() {
 			$this->addWhere( 'u1.user_editcount > 0' );
 		}
 
-		$user_groups = $this->fld_groups ? $this->getAllUserGroups() : array();
-		
 		if ( $this->fld_blockinfo ) {
-			$this->addTables('ipblocks');
-			$this->addTables('`user`', 'u2');
-			$u2 = sprintf("%s %s", '`user`', 'u2');
+			$this->addTables( 'ipblocks' );
+			$this->addTables( '`user`', 'u2' );
+			$u2 = sprintf( "%s %s", '`user`', 'u2' );
 			$this->addJoinConds(
 				array(
-					'ipblocks' => array('LEFT JOIN', 'ipb_user=u1.user_id'),
-					$u2 => array('LEFT JOIN', 'ipb_by=u2.user_id')
+					'ipblocks' => array( 'LEFT JOIN', 'ipb_user=u1.user_id' ),
+					$u2 => array( 'LEFT JOIN', 'ipb_by=u2.user_id' )
 				)
 			);
-			$this->addFields(array('ipb_reason', 'u2.user_name blocker_name'));
+			$this->addFields( array( 'ipb_reason', 'u2.user_name blocker_name' ) );
 		}
 
-		$this->addOption('LIMIT', $limit + 1);
+		$this->addOption( 'LIMIT', $limit + 1 );
 
 		$this->addFields  ( 'u1.user_name, u1.user_id' );
 		$this->addFieldsIf( 'u1.user_editcount', $this->fld_editcount );
 		$this->addFieldsIf( 'u1.user_registration', $this->fld_registration );
 
 		$this->addOption  ( 'ORDER BY', 'u1.user_name' );
 
-		$res = $this->select(__METHOD__);
+		$res = $this->select( __METHOD__ );
 
 		$data = array ();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$data" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)